docs-source/migrating.md

```diff
@@ -2,9 +2,7 @@
 
 ## Migrating from v6 to v7
```

Comment:
Do we want to keep this file around? Are there still people willing to migrate from v4, for example?

Reply:
I don't think there is any harm in keeping it around for now.